tests/account/test_EthAccount.py

```diff
@@ -0,0 +1,196 @@
+from signers import MockeEthSigner
+
+private_key = b'\x01' * 32
+signer = MockeEthSigner(b'\x01' * 32)
+other = MockeEthSigner(b'\x02' * 32)
+
+IACCOUNT_ID = 0xf10dbd44
+
+
+@pytest.fixture(scope='module')
+def contract_defs():
+    account_def = get_contract_def('openzeppelin/account/EthAccount.cairo')
+    init_def = get_contract_def("tests/mocks/Initializable.cairo")
+    attacker_def = get_contract_def("tests/mocks/account_reentrancy.cairo")
+
+    return account_def, init_def, attacker_def
+
+
+@pytest.fixture(scope='module')
+async def account_init(contract_defs):
+    account_def, init_def, attacker_def = contract_defs
+    starknet = await Starknet.empty()
+
+    account1 = await starknet.deploy(
+        contract_def=account_def,
+        constructor_calldata=[signer.eth_address]
+    )
+    account2 = await starknet.deploy(
+        contract_def=account_def,
+        constructor_calldata=[signer.eth_address]
+    )
+    initializable1 = await starknet.deploy(
+        contract_def=init_def,
+        constructor_calldata=[],
+    )
+    initializable2 = await starknet.deploy(
+        contract_def=init_def,
+        constructor_calldata=[],
+    )
+    attacker = await starknet.deploy(
+        contract_def=attacker_def,
+        constructor_calldata=[],
+    )
+
+    return starknet.state, account1, account2, initializable1, initializable2, attacker
+
+
+@pytest.fixture
+def account_factory(contract_defs, account_init):
+    account_def, init_def, attacker_def = contract_defs
+    state, account1, account2, initializable1, initializable2, attacker = account_init
+    _state = state.copy()
+    account1 = cached_contract(_state, account_def, account1)
+    account2 = cached_contract(_state, account_def, account2)
+    initializable1 = cached_contract(_state, init_def, initializable1)
+    initializable2 = cached_contract(_state, init_def, initializable2)
+    attacker = cached_contract(_state, attacker_def, attacker)
+
+    return account1, account2, initializable1, initializable2, attacker
+
+
+@pytest.mark.asyncio
+async def test_constructor(account_factory):
+    account, *_ = account_factory
+
+    execution_info = await account.get_eth_address().call()
+    assert execution_info.result == (signer.eth_address,)
+
+    execution_info = await account.supportsInterface(IACCOUNT_ID).call()
+    assert execution_info.result == (TRUE,)
+
+
+@pytest.mark.asyncio
+async def test_execute(account_factory):
+    account, _, initializable, *_ = account_factory
+
+    execution_info = await initializable.initialized().call()
+    assert execution_info.result == (0,)
+
+    _, hash, signature = await signer.send_transactions(account, [(initializable.contract_address, 'initialize', [])])
+
+    validity_info, *_ = await signer.send_transactions(account, [(account.contract_address, 'is_valid_signature', [hash, len(signature), *signature])])
+    assert validity_info.result.response[0] == TRUE
```

Comment:
Very nice! Should we also check that `is_valid_signature` reverts when it should?